src/Dialogs/Plane/PlanePolarDialog.cpp

```diff
@@ -132,15 +134,15 @@ PlanePolarWidget::Prepare(ContainerWindow &parent,
   DataFieldListener *listener = this;
   Add(std::make_unique<PolarShapeEditWidget>(plane.polar_shape, listener));
 
-  AddFloat(_("Reference Mass"), _("Reference mass of the polar"),
+  AddFloat(_("Reference Mass"), _("Reference mass of the polar."),
```

Comment:
Unrelated change, doesn't belong in this commit

Reply:
see below

Reply:
see below

Reply:
I don't understand, see below what?

Reply:
My previous comment "below" was: It is related. The commit/branch tries to clean up the masses mess and unclear code comments are a target.